pkg/server/helper.go

```diff
@@ -10,9 +10,102 @@ import (
 
 	"github.com/30x/enrober/pkg/apigee"
 	"github.com/30x/enrober/pkg/helper"
+	"os"
+	"strconv"
 )
 
-// Lets start by just making a new function
+func GeneratePTS(depBody deploymentPost, org string) (v1.PodTemplateSpec, error) {
+
+	tempURI := os.Getenv("DOCKER_REGISTRY_URL")
+	if tempURI == "" {
+		return v1.PodTemplateSpec{}, errors.New("No URI set")
+	}
+
+	cdir := os.Getenv("POD_CDIR")
+	if cdir == "" {
+		cdir = "10.1.0.0/16"
+	}
+
+	var tempPaths string
+	var intPort int
+	if depBody.Paths == nil {
+		//Make default paths
+		defaultPath := []EdgePath{
+			{
+				BasePath:      "/" + depBody.DeploymentName,
+				ContainerPort: "9000",
+			},
+		}
+		intPort = 9000
+		var err error
+		tempPaths, err = composePathsJSON(defaultPath)
+		if err != nil {
+			return v1.PodTemplateSpec{}, err
+		}
+	} else {
+		var err error
+		intPort, err = strconv.Atoi(depBody.Paths[0].ContainerPort)
+		tempPaths, err = composePathsJSON(depBody.Paths)
+		if err != nil {
+			return v1.PodTemplateSpec{}, err
+		}
+	}
+	tempK8sEnv, err := apigee.ApigeeEnvtoK8s(depBody.EnvVars)
+	if err != nil {
+		return v1.PodTemplateSpec{}, err
+	}
+	apiKeyEnv := v1.EnvVar{
```

Comment:
It's not strictly necessary but it was in Kiln's implementation and it would make it much easier to do pod to pod communication.